src/devices/Bmxx80/Bme680.cs

```diff
@@ -51,49 +58,248 @@ public Bme680(I2cDevice i2cDevice)
 
             _communicationProtocol = CommunicationProtocol.I2c;
             _controlRegister = (byte)Bme680Register.CTRL_MEAS;
+
+            SetDefaultConfiguration();
+        }
+
+        private Sampling _humiditySampling;
+
+        /// <summary>
+        /// Gets or sets the humidity sampling.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the <see cref="Sampling"/> is set to an undefined mode.</exception>
+        public Sampling HumiditySampling
+        {
+            get => _humiditySampling;
+            set
+            {
+                if (!Enum.IsDefined(typeof(Sampling), value))
```

Comment:
Also wasn't sure if these setter should throw exceptions or just retain their value if an exception would be thrown.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/property states:

> DO preserve the previous value if a property setter throws an exception. 

Just wasn't sure if these rules are used for this repo.